File: stack/Stack.py
class Stack:
    def __init__(self) -> None:
        self.container=[]
    
    def push(self,item):
        self.container.append(item)
        return item
    def pop(self):
        if(len(self.container) ==0):
            return -1
        else:
            lastIndex = len(self.container) -1
            self.container.remove(self.container[lastIndex])
    
    def isEmpty(self)->bool:
        if(len(self.container)!=0):
            return False
        else:
            return True 
     
    def peek(self):
        if(len(self.container)!=0):
            lastIndex = len(self.container)-1               
            return self.container[lastIndex]
    def print(self):
        for item in range(0,len(self.container)):
                print(self.container[item])
                
                
                
                
                
stack = Stack()
stack.push(20)                
stack.push(24)                
stack.push(38)                
stack.push(45)                     
stack.push(56)
print("All pushed element")
stack.print()
print("After Poping element")
stack.pop()
stack.pop()
stack.print()                
print("isEmpty:",stack.isEmpty())
print("Peek element :",stack.peek())

File: stack/Stack.js
class Stack {
  constructor() {
    // creating empty array
    this.stack = [];
   
  }
  push(item) {
    this.stack.push(item);
    return item;
  }
  pop() {
    return this.stack.pop();
  }
  isEmpty() {
    if (!this.stack.length) return true;
    else return false;
  }
  peek() {
   return this.stack[this.stack.length - 1];
  }
  print() {
    this.stack.forEach((elem) => {
      console.log(elem);
    });
  }
}

const stack = new Stack();

stack.push(24);
stack.push(38);
stack.push(45);
stack.push(56);
console.log("All pushed element");
stack.print();
console.log("After Poping element");
stack.pop();
stack.pop();
stack.print();
console.log("isEmpty:", stack.isEmpty());
console.log("Peek element :", stack.peek());
